src/app/(main)/aboutus/page.tsx

```diff
@@ -0,0 +1,196 @@
+		{ name: 'David Cramer', role: 'Founder & CTO, Sentry', image: 'david-cramer' },
+		{ name: 'Timi Ajiboye', role: 'Founder & CEO, Helicarrier', image: 'timi-ajiboye' },
+		{ name: 'Tomiwa Lasebikan', role: 'Founder & CPO, Helicarrier', image: 'tomiwa-lasebikan' },
+		{ name: 'Arup Chakrabarti', role: 'Ex-Director of Engineering, PagerDuty', image: 'arup-chakrabarti' },
+		{ name: 'Prosper Otemuyiwa', role: 'Senior Developer Advocate, Novu', image: 'prosper-otemuyiwa' },
+		{ name: 'Fernando Mayo', role: 'Director of Engineering, Datadog', image: 'fernando-mayo' },
+		{ name: 'Kareem Kouddous', role: 'Ex-Engineering Leader, Airbnb', image: 'kareem-kouddous' }
+	];
+
+	const teamMembers = [
+		{
+			name: 'Raymond Tukpe',
+			role: 'Chief Technology Officer',
+			image: 'employee',
+			github: 'https://github.com/jirevwe',
+			linkedIn: 'https://www.linkedin.com/in/rtukpe/',
+			twitter: 'https://twitter.com/rtukpe'
+		},
+		{
+			name: 'Subomi Oluwalana',
+			role: 'Founder / C.E.O',
+			image: 'employee',
+			github: 'https://github.com/subomi',
+			linkedIn: 'https://www.linkedin.com/in/subomi-oluwalana-one/',
+			twitter: 'https://twitter.com/subomiOluwalana'
+		},
+		{
+			name: 'Emmanuel Aina',
+			role: 'Founder',
+			image: 'employee',
+			github: 'https://github.com/horlah',
+			linkedIn: 'https://www.linkedin.com/in/emmanuel-aina-o/',
+			twitter: 'https://twitter.com/emmanuelaina_'
+		},
+		{
+			name: 'Daniel Oluojomu',
+			role: 'Backend Engineer',
+			image: 'employee',
+			github: 'https://github.com/danvixent',
+			linkedIn: 'https://www.linkedin.com/in/daniel-oluojomu/',
+			twitter: 'http://twitter.com/danvixent'
+		},
+		{
+			name: 'Pelumi Oni',
+			role: 'Frontend Engineer',
+			image: 'employee',
+			github: 'https://github.com/oluwadaminiola',
+			linkedIn: 'https://www.linkedin.com/in/pelumioni',
+			twitter: 'https://twitter.com/pelumi__o'
+		},
+		{
+			name: 'Philip Simbwa',
+			role: 'Cloud Engineer',
+			image: 'employee',
+			github: 'https://github.com/simbwa',
+			linkedIn: 'https://www.linkedin.com/in/phillip-simbwa-8517741b/',
+			twitter: 'http://twitter.com/simbwa'
+		}
+	];
+
+	return (
+		<main>
+			<section className="max-w-[1062] flex flex-col justify-center items-center pt-220px pb-100px desktop:pb-236px px-20px">
+				<div className="bg-primary-25 rounded-[40px] w-fit py-10px px-20px flex items-center text-14 mb-40px">
+					We are backed by
+					<Image src="/svg/y-combinator.svg" width={24} height={24} className="ml-16px w-24px" alt="y-combinator" />
+				</div>
+				<h1 className="text-center desktop:text-[50px] desktop:leading-[80px] font-bold max-w-[910px] m-auto">Building the webhooks infrastructure for the internet.</h1>
+				<p className="text-center max-w-[950px] mx-auto text-16 leading-8 mobile:text-14 mt-16px text-gray-600">
+					We’re a team of fine engineers building the finest webhooks platform that engineers and teams of all sizes can rely on at scale.
```

Comment:
@Oluwadaminiola We’re a team of fine engineers building the finest webhooks infrastructure that engineers and teams of all sizes can rely on at scale.